src/core/packages/http/server-internal/src/logging/get_response_log.ts

```diff
@@ -59,23 +59,36 @@ export function getEcsResponseLog(request: Request, log: Logger) {
   const query = querystring.stringify(request.query);
   const pathWithQuery = query.length > 0 ? `${path}?${query}` : path;
 
-  // eslint-disable-next-line @typescript-eslint/naming-convention
-  const status_code = isBoom(response) ? response.output.statusCode : response.statusCode;
-
   const requestHeaders = cloneAndFilterHeaders(request.headers);
-  const responseHeaders = cloneAndFilterHeaders(
-    isBoom(response) ? (response.output.headers as HapiHeaders) : response.headers
-  );
 
   // borrowed from the hapi/good implementation
   const responseTime = (request.info.completed || request.info.responded) - request.info.received;
-  const responseTimeMsg = !isNaN(responseTime) ? ` ${responseTime}ms` : '';
+  const responseTimeMsg = !isNaN(responseTime) && responseTime >= 0 ? ` ${responseTime}ms` : '';
```

Comment:
```suggestion
  const responseTimeMsg = responseTime >= 0 ? ` ${responseTime}ms` : '';
```

I think the `isNaN` check is redundant if we are doing the `>= 0`

Reply:
True! didn't notice it, thanks!